common/environment/environment.cpp

```diff
@@ -913,6 +932,31 @@ class CConstInstanceInfo : public CConstEnvBase, implements IConstInstanceInfo
 
 };
 
+class CConstDropZoneServerInfo : public CConstEnvBase, implements IConstDropZoneServerInfo
+{
+public:
+    IMPLEMENT_IINTERFACE;
+    IMPLEMENT_ICONSTENVBASE;
+    CConstDropZoneServerInfo(CLocalEnvironment *env, IPropertyTree *root) : CConstEnvBase(env, root), prop(root) {}
+
+    virtual StringBuffer & getName( StringBuffer & name) const
+    {
+        name.set(prop->queryProp("@name"));
+        return name;
+    }
+    virtual StringBuffer & getServer( StringBuffer & server) const
+    {
+        server.set(prop->queryProp("@server"));
```

Comment:
the convention on StringBuffer &get* methods is to append to the buffer, so that callers can fill up a buffer as they see fit.
i.e.:
```
    virtual StringBuffer & getServer( StringBuffer & server) const
    {
        return server.append(prop->queryProp("@server"));
    }
```